pkg/cmd/package/nuget/create/create.go

```diff
@@ -0,0 +1,315 @@
+	flags.StringVar(&createFlags.Description.Value, createFlags.Description.Name, "", "A description of the package")
+	flags.StringVar(&createFlags.ReleaseNotes.Value, createFlags.ReleaseNotes.Name, "", "Release notes for this version of the package")
+	flags.StringVar(&createFlags.ReleaseNotesFile.Value, createFlags.ReleaseNotesFile.Name, "", "A file containing release notes for this version of the package")
+	flags.SortFlags = false
+
+	return cmd
+}
+
+func createRun(opts *NuPkgCreateOptions) error {
+	if !opts.NoPrompt {
+		if err := pack.PackageCreatePromptMissing(opts.PackageCreateOptions); err != nil {
+			return err
+		}
+		if err := PromptMissing(opts); err != nil {
+			return err
+		}
+
+	}
+
+	if opts.Id.Value == "" {
+		return errors.New("must supply a package ID")
+	}
+
+	err := applyDefaultsToUnspecifiedPackageOptions(opts)
+	if err != nil {
+		return err
+	}
+
+	nuspecFilePath := ""
+	if shouldGenerateNuSpec(opts) {
+		nuspecFilePath, err = GenerateNuSpec(opts)
+		if err != nil {
+			return err
+		}
+		opts.Include.Value = append(opts.Include.Value, opts.Id.Value+".nuspec")
+	}
+
+	pack.VerboseOut(opts.Verbose.Value, "Packing \"%s\" version \"%s\"...\n", opts.Id.Value, opts.Version.Value)
+	outFilePath := pack.BuildOutFileName("nupkg", opts.Id.Value, opts.Version.Value)
+
+	err = pack.BuildPackage(opts.PackageCreateOptions, outFilePath)
+	if err != nil {
+		return err
+	}
+
+	if !opts.NoPrompt {
+		autoCmd := flag.GenerateAutomationCmd(
+			opts.CmdPath,
+			opts.Author,
+			opts.Title,
+			opts.Description,
+			opts.ReleaseNotes,
+			opts.ReleaseNotesFile,
+			opts.Id,
+			opts.Version,
+			opts.BasePath,
+			opts.OutFolder,
+			opts.Include,
+			opts.Verbose,
+			opts.Overwrite,
+		)
+		fmt.Fprintf(opts.Writer, "\nAutomation Command: %s\n", autoCmd)
+	}
+
+	if nuspecFilePath != "" {
+		return os.Remove(nuspecFilePath)
+	}
+
+	return nil
+}
+
+func PromptMissing(opts *NuPkgCreateOptions) error {
+	if len(opts.Author.Value) == 0 {
+		message := "Author"
+		for {
+			var author string
+			if err := opts.Ask(&survey.Input{
+				Message: message,
+				Help:    "Add an author to the package metadata; if no authors are specified the author will default to the current user.",
+			}, &author); err != nil {
+				return err
+			}
+			if author == "" {
+				break
+			}
+			opts.Author.Value = append(opts.Author.Value, author)
+			message = message + " (leave blank to continue)"
+		}
+	}
+
+	if opts.Title.Value == "" {
+		if err := opts.Ask(&survey.Input{
+			Message: "Nuspec title",
+			Help:    "The title to include in the Nuspec file.",
+		}, &opts.Title.Value); err != nil {
+			return err
+		}
+	}
+
+	if opts.Description.Value == "" {
+		if err := opts.Ask(&survey.Input{
+			Message: "Nuspec description",
+			Help:    "The description to include in the Nuspec file; defaults to \"A deployment package created from files on disk.\".",
+		}, &opts.Description.Value); err != nil {
```

Comment:
set the `Default` property to the default value

Reply:
should not set this one. Removed the text as it was never set. If we did set this as default nuspec file will try to generate all the time when it does not need too.

Reply:
Agreed, but we could set it an author has been provided, since author is mandatory, but not important

Reply:
agreed, I will also change the prompts so it only asks the other nuspec related question if you provide an author.